docs/http/http-spans.md

```diff
@@ -66,27 +66,27 @@ and various HTTP versions like 1.1, 2 and SPDY.
 
 ## Name
 
+**Status**: [Mixed][DocumentStatus]
+
 HTTP spans MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/trace/api.md#span).
 
+HTTP span names SHOULD be `{method} {target}` if there is a (low-cardinality) `target` available. If there is no (low-cardinality) `{target}` available, HTTP span names SHOULD be `{method}`.
+
 <!-- markdown-link-check-disable -->
 <!-- HTML anchors are not supported https://github.com/tcort/markdown-link-check/issues/225-->
-HTTP server span names SHOULD be `{method} {http.route}` if there is a
-(low-cardinality) `http.route` available (see below for the exact definition of the [`{method}`](#method-placeholder) placeholder).
-
-If there is no (low-cardinality) `http.route` available, HTTP server span names
-SHOULD be [`{method}`](#method-placeholder).
-
-HTTP client spans have no `http.route` attribute since client-side instrumentation
-is not generally aware of the "route", and therefore HTTP client span names SHOULD be
-[`{method}`](#method-placeholder).
+(see below for the exact definition of the [`{method}`](#method-placeholder) and [`{target}`](#target-placeholder) placeholders).
 <!-- markdown-link-check-enable -->
 
 The <span id="method-placeholder">`{method}`</span> MUST be `{http.request.method}` if the method represents the original method known to the instrumentation.
 In other cases (when `{http.request.method}` is set to `_OTHER`), `{method}` MUST be `HTTP`.
 
-Instrumentation MUST NOT default to using URI
-path as span name, but MAY provide hooks to allow custom logic to override the
-default span name.
+The <span id="target-placeholder">`{target}`</span> SHOULD be one of the following:
+
+- [`http.route`](/docs/attributes-registry/http.md) for HTTP Server spans
+- [`url.template`](/docs/attributes-registry/url.md) if enabled and available (![Experimental](https://img.shields.io/badge/-experimental-blue))
```

Comment:
```suggestion
- [`url.template`](/docs/attributes-registry/url.md) for HTTP Client spans if enabled and available (![Experimental](https://img.shields.io/badge/-experimental-blue))
```

Reply:
I intentionally left span kind unspecified here.
The motivation: if HTTP server framework doesn't have a concept of a route, one could write a regex and extract a template. It might belong in a `url.template` than `http.route`. So I wanted us to leave a room to apply `url.template` to server spans as well.

I'm also ok following the suggestion and limiting `url.template`  to client spans only